lib/auth/auth_with_roles.go

```diff
@@ -240,18 +251,23 @@ func hasLocalUserRole(checker services.AccessChecker) bool {
 }
 
 // CreateSessionTracker creates a tracker resource for an active session.
-func (a *ServerWithRoles) CreateSessionTracker(ctx context.Context, req *proto.CreateSessionTrackerRequest) (types.SessionTracker, error) {
-	if !a.hasBuiltinRole(string(types.RoleKube)) && !a.hasBuiltinRole(string(types.RoleNode)) && !a.hasBuiltinRole(string(types.RoleProxy)) {
-		return nil, trace.AccessDenied("this request can be only executed by a node, proxy or kube service")
+func (a *ServerWithRoles) CreateSessionTracker(ctx context.Context, tracker types.SessionTracker) (types.SessionTracker, error) {
+	if a.hasBuiltinRoleInList(types.RoleKube, types.RoleNode, types.RoleProxy, types.RoleDatabase, types.RoleApp, types.RoleWindowsDesktop) {
```

Comment:
Should we capture this set of roles in a global var and reuse it? Ie, `var sessionTrackerRoles = ...`?

Ditto for the error message, but in a const?